.travis.yml

```diff
@@ -9,6 +9,9 @@ notifications:
   webhooks:
     - http://savage.nonblocking.io:8080/savage/travis
 before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Not a huge fan of abusing npm to install yarn? 😆

Reply:
I'm still playing around with this. Now that we recommend the use of `yarn 1.0.2` (latest stable) for development, I thought we should use the same versions on Travis too. Let me know if you can think of a more efficient way to install the correct version.

Reply:
This is one of the recommended ways to install the latest version of Yarn from yarkpkg.com, so might as well stick with it. I'm not advocating for one or the other.

I think the "better alternative" that yarkpkg.com recommends is to use the Debian packages, but it might slow down the installation (because it needs to add the ppa, update the apt packages list, and download and install yarn - as opposed to what you chose which just downloads a simple .sh file and installs it)

Reply:
I agree that we should choose the fastest install method for Travis, while recommending the "best" install method for local development. It's fine if the two aren't the same.

Reply:
:+1:

Reply:
Any issues with Travis's built-in version of yarn? How long does this take to install?

Reply:
It's prehistoric (0.27 I think)

Reply:
Travis uses `yarn 0.27.5`. Our packages install fine with that version. I thought we should eventually get to the point where everyone writing AMP code should use the same tool chain, but I'll do that in a separate PR.

I've reverted the changes in this file.